app/helpers/admissions_helper.rb

```diff
@@ -35,4 +35,12 @@ def facet_count_for_gender(value)
   def default_value(value)
     params[value]
   end
+
+  def friendly_date(value)
+    date_string = value.to_s
+    year = date_string[0..3]
+    month = date_string[4..5]
+    day = date_string[6..7]
+    "#{month}/#{day}/#{year}"
+  end
```

Comment:
Okay I tested them (mine is with the regex in the name):

```ruby
> Benchmark.bm do |x|
>   x.report { 100000.times do; friendly_date(19830101); end }
>   x.report { 100000.times do; friendly_date_regex(19830101); end }
> end
       user     system      total        real
   0.110000   0.000000   0.110000 (  0.114900)
   0.250000   0.000000   0.250000 (  0.261434)
```

The regex way is slower, an extra .15 seconds when you run it 100000 times!